src/App.jsx

```diff
@@ -1,13 +1,188 @@
-import { useState } from 'react'
-import './App.css'
+import React, { useCallback, useState } from "react";
+import ClassCheckBox from "./components/ClassCheckbox";
+import "./App.css";
 
-function App() {
+const classes = [
```

Comment:
We can store this data in a constant file.

Reply:
Okay I will create a separate constants file.